Makefile

```diff
@@ -1,9 +1,10 @@
 data/whitelisted_ext_idefs.json: 
 	python3 wikidatarefisland/run.py --step ss1 --output "whitelisted_ext_idefs.json"
-
+data/extracted_unreferenced_statements.jsonl: \
+	data/whitelisted_ext_idefs.json
+	python3 wikidatarefisland/run.py --step extract_items --side-service-input "whitelisted_ext_idefs.json" --input "dump.json" --output "extracted_unreferenced_statements.jsonl"
 data/scraped_data.jsonl:
-	python3 wikidatarefisland/run.py --step scrape --input "pipe1.jsonl" --output "scraped_data.jsonl"
-
+	python3 wikidatarefisland/run.py --step scrape --input "extracted_unreferenced_statements.jsonl" --output "scraped_data.jsonl"
```

Comment:
We should probably add `extracted_unreferenced_statements.jsonl` as a dependency here

Reply:
Yes! And I know you said this before. Seemingly lost in it in the repeated rebasing and merging :/ sorry

Reply:
No problem :) thanks for the change